manifests/postgresql-operator-default-configuration.yaml

```diff
@@ -91,4 +91,7 @@ configuration:
     # scalyr_api_key: ""
     # scalyr_image: ""
     # scalyr_server_url: ""
-
+  logical_backup_key:
```

Comment:
why is it called "_key"? I'm missing the enable_logical_backup flag here.

Reply:
> why is it called "_key"?

by mistake, will fix

> I'm missing the enable_logical_backup flag here. Same for configMap

There is no such key, because that particular feature is supposed  to be triggered by users only on very few selected PG clusters  ; it hardly makes sense to enable it for all DBs within a k8s cluster.